pkg_specs/bitcoin-@variant.sps

```diff
@@ -49,6 +49,14 @@ The full path to the directory which will contain timechain data (blocks and cha
 Important: you need around 10GB of free space!
 """
 
+[config."bitcoin.conf".ivars.debuglogfile]
+type = "path"
+file_type = "regular"
+create = { mode = 755, owner = "$service", group = "$service" }
```

Comment:
Better let bitcoind create it, we don't need to. (IOW, delete this line; note that mode would've been incorrect too)
However the parent should be created, ideally by systemd - add:

```
LogsDirectory=bitcoin-{variant}
```

to `extra_service_config`